pom.xml

```diff
@@ -37,25 +35,25 @@
     <dependency> <!-- TODO switch to MockAuthorizationStrategy -->
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-auth</artifactId>
-      <version>2.2</version>
+      <version>3.1.7</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>mock-slave</artifactId>
-      <version>1.11</version>
+      <version>125.vcfb_5c627d399</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.test</groupId>
       <artifactId>docker-fixtures</artifactId>
-      <version>1.5</version>
+      <version>166.v912b_95083ffe</version>
```

Comment:
was so old the `apt` repositories had long gone so the container would not build.